### PR TITLE
OE-6551 - Save prescriptions without printing

### PR DESCRIPTION
**Pull request by edgaaar8**

### CODE REVIEW

**Comment by biskyt** [comment]
@splatEric would you be so kind as to review and merge. Thanks

File: protected/modules/OphDrPrescription/assets/js/module.js [hunk 1]
```diff
@@ -39,6 +39,12 @@ $(document).ready(function () {
       e.preventDefault();
     }
   });
+    handleButton($('#et_save'), function (e) {
+        $('#Element_OphDrPrescription_Details_draft').val(0);
```
Comment (splatEric) [inline protected/modules/OphDrPrescription/assets/js/module.js]: Should this value change only happen if the prescription has length?

Comment (edgaaar8) [inline protected/modules/OphDrPrescription/assets/js/module.js]: There is a check for a blank prescription in the code
`if(!checkPrescriptionLength()){`
and we've got [this](https://lh3.googleusercontent.com/-80OXcE9FOeI/WVuOULJVtEI/AAAAAAAAB_k/Bhi3S4zl9kUrgdK5onoD-XRyBE8MI6D4wCL0BGAYYCw/h532/2017-07-04.png) while were trying to save an empty prescription

Comment (splatEric) [inline protected/modules/OphDrPrescription/assets/js/module.js]: I guess my concern is that it is changing the draft status whether it submits or not. It just seemed a little odd, but it's actually fine so long as all the buttons that might be clicked to cause a save will change that value first.

**Comment by biskyt** [comment]
Been merged into develop

**Pull request closed without merging**